Aboutme.md

```diff
@@ -0,0 +1,19 @@
+### :open_file_folder: What have I learnt?
+* The different principles of Software Development.
+* How to compile a Java
+* The different variables in Java
+
+
+### :confused: What am I confused about?
+* I am not confused about anything, I just need a practical example of SOLID principle of software development. I think once the course progresses I will better understand this. At the moment it's just an abstaract concept.
```

Comment:
typo on  the word "abstract". We'll look at creating some code examples of SOLID, but you're right as the course progresses you'll get a better understading of these in practice.